app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
```

Comment:
The `people` dictionary is defined as a class attribute, which means it is shared across all instances of the `Person` class. This can lead to issues if you create multiple `Person` objects, as they will all share the same `people` dictionary. Consider using an instance attribute instead if you want each `Person` to have its own dictionary.